bitrise.yml

```diff
@@ -1608,3 +1621,6 @@ trigger_map:
   pipeline: pipeline_multiple_shards
 - pull_request_target_branch: release/v*
   pipeline: pipeline_multiple_shards
+- pull_request_target_branch: main
```

Comment:
this is not correctly set, that's why it does not work. It should be just:
- pull_request_target_branch: main
   workflow: Bitrise_Performance_Test

Reply:
you may need to change also the push to target branch